app/react/V2/Components/UI/Table/TableBody.tsx

```diff
@@ -48,6 +49,7 @@ const setRowId: <T>(
 // eslint-disable-next-line comma-spacing
 const TableBodyComponent = <T,>({
   draggableRows,
+  allowEditGroupsWithDnD,
   // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
I think this can be removed

Reply:
I checked it and the property is needed to initialize the DnDOperations settings that are needed [later](https://github.com/huridocs/uwazi/blob/6715-thesauri-screen-crash-on-dnd-rows/app/react/V2/Components/Layouts/DragAndDrop/useDnDContext.tsx#L27) in the draggable items. So I'm keeping it.